src/Interpreters/Session.cpp

```diff
@@ -401,13 +406,20 @@ ContextMutablePtr Session::makeSessionContext()
     prepared_client_info.reset();
 
     /// Set user information for the new context: current profiles, roles, access rights.
+    /// TCP handler quota is received after context creation.
```

Comment:
`/// In the TCP handler, the quota key is received in "addendum packet" after context creation --> skip initialization`


(same in l. 458)

Reply:
Done